application/forms/Command/Instance/ToggleInstanceFeaturesForm.php

```diff
@@ -37,6 +45,61 @@ public function __construct($featureStatus)
         ];
 
         $this->getAttributes()->add('class', 'instance-features');
+
+        $this->on(self::ON_SUCCESS, function () {
+            /** @var ToggleInstanceFeatureCommand $feature */
```

Comment:
Is this necessary with the updated type-hint above?

Reply:
No.